modules/ui/fields/input.js

```diff
@@ -404,16 +407,19 @@ export function uiFieldText(field, context) {
         }
     }
 
-
     function change(onInput) {
         return function() {
             var t = {};
             var val = utilGetSetValue(input);
-            if (!onInput) val = context.cleanTagValue(val);
 
-            // don't override multiple values with blank string
+            if (field.type === 'tel') {
+                val = val.replace(/[^0-9+]/g, '');
```

Comment:
This appears to be stripping anything that isn’t a digit or plus sign from the tag value. I don’t think this is consistent with what the `phone` key is supposed to be set to in most regions, at least according to [this documentation](https://wiki.openstreetmap.org/wiki/Key:phone). The linked issue appears to be more limited in scope: it would only take effect if the field’s value resembles a tel: URI. But tel: URIs can certainly contain non-digits.

Reply:
Yes, I wasn't aware of that previously. So, what specifically should be disregarded?

Reply:
If I understand correctly, https://github.com/openstreetmap/iD/issues/9559#issue-1634183847 was only asking to strip the `tel:` or `tel://` prefix if present.